app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::BooksController < Api::V1::BaseController
+  before_action :load_book, only: :show
+
+  def index
+    books = Book.all
+
+    return render json: books, each_serializer: BookSerializer
```

Comment:
``
```suggestion
    render json: Book.all
```
el return es implicito, y el serializer tambien por tener el mismo nombre de clase

Reply:
Impresionante, cambiado!